<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtSupport::Internal::QtVersionManager</class>
 <widget class="QWidget" name="QtSupport::Internal::QtVersionManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>450</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeView" name="qtdirList">
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Utils::DetailsWidget" name="versionInfoWidget" native="true"/>
     </item>
     <item>
      <widget class="Utils::DetailsWidget" name="infoWidget" native="true"/>
     </item>
     <item>
      <widget class="Utils::DetailsWidget" name="debuggingHelperWidget" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="minimumSize">
        <size>
         <width>21</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Add...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delButton">
       <property name="minimumSize">
        <size>
         <width>21</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cleanUpButton">
       <property name="text">
        <string>Clean Up</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>77</width>
         <height>37</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::DetailsWidget</class>
   <extends>QWidget</extends>
   <header location="global">utils/detailswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>qtdirList</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>delButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
